README.md

```diff
@@ -61,13 +61,19 @@ Package [samza.examples.cookbook](https://github.com/apache/samza-hello-samza/tr
 Package [samza.examples.wikipedia.application](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/wikipedia/application) contains a small Samza application which consumes the real-time feeds from Wikipedia, extracts the metadata of the events, and calculates statistics of all edits in a 10-second window. You can start the app on the grid using the run-app.sh script:
 
 ```
-./deploy/samza/bin/run-app.sh --config-factory=org.apache.samza.config.factories.PropertiesConfigFactory --config-path=file://$PWD/deploy/samza/config/wikipedia-application.properties
+./deploy/samza/bin/run-app.sh \
+  --config app.class=samza.examples.wikipedia.application.WikipediaApplication \
+  --config yarn.package.path=file:///Users/kwu/workspace/hello-samza/target/hello-samza-1.5.0-SNAPSHOT-dist.tar.gz \
+  --config job.name=wikipedia-application \
+  --config job.factory.class=org.apache.samza.job.yarn.YarnJobFactory \
+  --config job.config.loader.factory=org.apache.samza.config.loaders.PropertiesConfigLoaderFactory \
+  --config job.config.loader.properties.path=$PWD/deploy/samza/config/wikipedia-application.properties
```

Comment:
There are several other places which did not have these additional configs (e.g. `run-event-hubs-zk-application.sh`, `CouchbaseTableExample.java`). Do those places need to be updated also?

Reply:
Yes.